Gemfile.jruby-2.3.lock.release

```diff
@@ -186,7 +186,7 @@ GEM
       rspec (~> 3.0)
       rspec-wait
       stud (>= 0.0.20)
-    logstash-filter-aggregate (2.6.3)
+    logstash-filter-aggregate (2.7.0)
```

Comment:
Seems like a good and safe change: https://github.com/logstash-plugins/logstash-filter-aggregate/blob/master/CHANGELOG.md#270